pkg/network/setup/netstat.go

```diff
@@ -103,11 +103,11 @@ func (c *NetStat) UpdateStatus(vmi *v1.VirtualMachineInstance, domain *api.Domai
 		sriovIfacesStatusFromDomainHostDevices(domain.Spec.Devices.HostDevices, vmiInterfacesSpecByName)...,
 	)
 
-	// TODO this is needed since currently passt is configured directly via qemu so the passt interfaces are not in the libvirt domain
-	// once passt will be configured via libvirt, this code can be removed
-	interfacesStatus = append(interfacesStatus,
-		passtIfacesStatusFromVmiSpec(vmi.Spec.Domain.Devices.Interfaces)...,
-	)
+	//// TODO this is needed since currently passt is configured directly via qemu so the passt interfaces are not in the libvirt domain
+	//// once passt will be configured via libvirt, this code can be removed
+	//interfacesStatus = append(interfacesStatus,
+	//	passtIfacesStatusFromVmiSpec(vmi.Spec.Domain.Devices.Interfaces)...,
+	//)
```

Comment:
Shouldn't this block be dropped completely instead of commented out?

Reply:
Yes, my bad, I will remove it altogether.